orchagent/portsorch.cpp

```diff
@@ -3378,6 +3549,336 @@ bool PortsOrch::removeAclTableGroup(const Port &p)
     return true;
 }
 
+
+sai_object_id_t PortsOrch::getStpInstanceOid(sai_uint16_t stp_instance)
```

Comment:
Adding STP functions to portsorch has considerably increased the file size. I think STP functions should be handled separately and not overload portsorch. Like an `stporch.cpp`

Reply:
Hi Prince, Considering STP has limited data and is part of vlan and port we included the code as part of portsorch. This has undergone extensive testing and might be risky to change at this point of time. We have additional spanning-tree features in the next release as part of which we will move this to separate file, hope thats acceptable. Thanks.

Reply:
I tend to disagree. Portsorch is not intended to handle protocol aspects and this is an overload. It is better to address/finalize this now than a later churn.

Reply:
Separated STP changes from portsorch. Please review the latest commit/changes